Kristea-Gamel

bykristea opened the pull request:
initial commit

bykristea committed:
Kristea-Gamel

initial commit

bykristea committed:
began setting up componend in header and imports to header and footer

bykristea committed:
added in images to each component

bykristea committed:
found example code to add to footer

added in functionality to footer icons to allow to heart un heart, retweet, open mail and to say comments comment ability coming soon.

bykristea committed:
added in moment.js

to diplay the date on the actual day you are looking at it

bykristea committed:
Setting Up Project-II-Calculator

bykristea committed:
got calculator to show up

styled buttons and used props to get the visual there

bykristea committed:
edited Button.css

tweaked the height of the app and the operators column.

bykristea committed:
didn’t mean to roll back commit on footer.

bykristea committed:
Merge branch 'Kristea-Gamel' of https://github.com/bykristea/React-UI-Components into Kristea-Gamel

bykristea commented on Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js:
```diff
@@ -1,19 +1,64 @@
-        </div>
-    );
-};
+class Footer extends React.Component {
```
Found some example code using props and state. I decided to go back to original because i didn't quite get it. I want to read up more on that tomorrow. went back to the lines 4-17

bykristea committed:
deleting example code

theryanclausen reviewed (commented):
Oops. Looks like you installed moment outside of your react app on a file level where there wasn't a git ignore for node modules. That would explain the 402 file changes and why your timestamp doesn't work. Make sure you are at the file level from which you `yarn start` when you `yarn add` anything or your react app won't have access to it.

bykristea committed:
buttons will register

got buttons to register and display. Operators are not working. Clear and zero are

bykristea committed:
edited css files to tweak widths and paddings

also tweaked font weight

bykristea commented:
I believe that I tweaked moment js in another commit. Because when I open the react project the date does show change to today's date. 

![image 3-2-19 at 6 04 pm](https://user-images.githubusercontent.com/9545600/53689195-b57d3500-3d15-11e9-985f-f187856f342c.jpg)

theryanclausen reviewed (approved):
Moment is working on your local machine because you have access to your entire file structure locally. When implementing any sort of deployment, your react app will not have access to any dependencies below the level of the folder in which the react app is initiated. Here is a link to a code sandbox generated from your social card. https://codesandbox.io/s/github/bykristea/React-UI-Components/tree/Kristea-Gamel/Project-I-Social-Card/social-card 

It's best get in the habit of adding all dependencies for a given react project in the same level as your react app package json. When you see `node_modules` in your repo, that means you have run an install outside of the scope of your react app, because the react app will generate a file that tells github to ignore the node_modules created at that level. Pushing node module files is considered a bad practice. People cloning your work to their machine should be creating their own node modules with a yarn install. 

Your Calculator is looking good. Nice job getting a start on your functionality. Great job organizing your file structure.
